opentelemetry-sdk/src/opentelemetry/sdk/trace/export/__init__.py

```diff
@@ -228,7 +230,10 @@ def on_end(self, span: ReadableSpan) -> None:
             if not self._spans_dropped:
                 logger.warning("Queue is full, likely spans will be dropped.")
                 self._spans_dropped = True
-
+                self._dropped_counter = Meter.create_counter(
```

Comment:
Take a look at how this is being implemented for the [Flask instrumentation](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/instrumentation/opentelemetry-instrumentation-flask/src/opentelemetry/instrumentation/flask/__init__.py#L497).